### DISCUSSION TITLE
社交機器人Moxie－輔助兒童學習🤖(so cute)

### DISCUSSION THREAD

**Discussion by bonny123865** [1101-GS3073-C/Python-Community-GS3073-C#84] [General]
**新聞內容:**
焦慮、過動症、自閉症的治療費極高且醫療資源不足，在發展或多或少都會遭遇困難。目前已存在許多支援STEM(科學、技術、工程、數學)教育的工具，但情緒智商輔助工具卻相對稀少。
美國加州Pasadena的Embodied Inc.日前發表社交機器人Moxie，專為**促進兒童認知、情緒、社交發展**而設計。善用電腦視覺與對話式人工智慧(Conversational AI)技術，強化與輔助而非取代兒童的正規治療。
Moxie運用**機器學習**(machine learning)與SocialX平台感知與互動，結合自然語言處理(NLP)、分析意圖與表情，透過有感染力的言語以及臉部表情與結合肢體動作來理解與表達情緒，並從心理學與神經學層面跟兒童建立更深聯結。另外提供Global Robotics Laboratory入口網站讓兒童掌握自己的進展。
Moxie透過跟兒童介紹使用介面與一般性指令，約需數週來熟悉個別兒童使用者，確保能維持良好互動，包括自動辨識適當的說話與停止發言時機。Moxie可藉由多模融合，從視覺與語音的來源方向聚焦互動的對象，並運用基於視覺的**同步定位與地圖建構**(VSLAM)技術解決導航問題。
**新聞心得:**
心理疾病的治療相對於身理方面模糊許多，這台社交機器人的出現就像是給兒童一個學習的小玩伴一樣，而且有面部表情、說話會辨識時機點感覺更有人性化！它集結了多項科技技術，從感知到分析到導航到輔助兒童發展，像是一個多元化的學習助理，而它的使用可以還能讓兒童自主掌握進度，相對平易近人。（造型有那麼一點像瓦力裡面的伊芙）
**新聞來源:**
https://www.digitimes.com.tw/iot/article.asp?cat=158&cat1=20&cat2=&id=0000584451_LFU5U6LW45XLF92XZBOM9
![embodied-moxie-robot-technology_dezeen_2364_col_5-852x479](https://user-images.githubusercontent.com/90883526/138886158-41048bea-9b10-4cbe-845a-21b3e04bf4e8.jpg)

**Comment by Jay-722** [comment]
挖賽，機器人越來越猛，這種要跟有自閉焦慮過動的小孩相處並給予幫助是很困難的任務

**Comment by wh20502** [comment]
我記得如果家裡有養小狗或寵物，小孩子跟寵物多溝通或是多陳述內心情緒的話，長大以後也會比較有自信，不知道是否AI也能具有相同的效果，或是小孩會認為那是假的也說不定。

**Comment by Shiba010** [comment]
現在的家長都因為工作繁忙，比較沒時間陪伴孩童。希望能透過這項技術讓小朋友比較不孤單。

**Comment by jack90325** [comment]
很多心理疾病患者會不想接觸人，所以這個發明確實是可以幫助醫師了解患者病況

**Comment by BingRong1217** [comment]
感覺對很多心理疾病患者會有很大的幫助

**Comment by Ben6uo** [comment]
但會不會因為流行了一些危險的活動而推薦給小朋友?

**Comment by HsuChen88** [comment]
若我小時候也有這個，也會有很大的幫助~

**Comment by larry40305** [comment]
這種機器人能代替忙碌的家長陪伴小孩，尤其現在人越來越忙碌

**Comment by liangyinglly** [comment]
對於家中只有一個小孩的家庭，會相當適合使用這個機器人，他除了能陪伴也能關心到小孩的情緒與狀況

**Comment by Vincent20030314** [comment]
這項發明確實能幫助自閉症患者和沒有人陪的小孩。

**Comment by yjpxi** [comment]
我覺得雖然Moxie旨在輔助而非取代正規治療，但長期依賴機器人互動可能會減少兒童與同伴或家長的實際社交機會，可能導致兒童在現實生活中缺乏與他人溝通的實際經驗，進而影響他們的社交能力。
